sandbox-web/src/main/java/com/exadel/sandbox/controllers/location_controller/LocationController.java

```diff
@@ -42,9 +41,10 @@ public LocationController(LocationService locationService, CountryService countr
             consumes = {"application/json"},
             path = "/newLocation")
     public ResponseEntity<?> createLocation(@RequestBody final LocationRequest locationRequest) {
```

Comment:
Why not to split to separate controllers: country management and location management?